log/log.go

```diff
@@ -0,0 +1,23 @@
+package log
+
+import (
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+func InitLogger(logLevel string) {
+
+	var zapLogLevel zapcore.Level
+
+	switch logLevel {
+	case "debug":
+		zapLogLevel = zap.DebugLevel
+	case "info":
+		zapLogLevel = zap.InfoLevel
```

Comment:
I'd make info default to make it bulletproof

Reply:
Sure, will do